liblepton/lib/geda-system.conf

```diff
@@ -135,3 +136,15 @@ monochrome=false
 paper=iso_a4
 size=
 
+
+
+#
+# liblepton:
+#
+
+[library]
+attribute-promotion=true
+promote-invisible=false
+keep-invisible=true
+always-promote-attributes=footprint;device;value;model-name
```

Comment:
@graahnul-grom,
Since the new config keys are related to `lepton-schematic` only, I would suggest to add two new groups: `[schematic.attribute]` and `[schematic.backup]` and move the keys there. Then we could shorten the key names, say, `attribute-promotion` -> `promotion`, `always-promote-attributes` -> `always-promote`, `make-backup-files` -> something like `save-files` or `enable-save`, or even reverse the logic and use just `disable` to disable backups (seems even better IMO).

Reply:
Maybe [schematic.attribute**s**]?
```ini
[schematic.attributes]
promote=true
always-promote=footprint;device;value;model-name
promote-invisible=false
keep-invisible=true
```

Logic inversion in settings could be quite confusing, IMHO.
```ini
[schematic.backup]
create-files=true
```

Reply:
... or just `[schematic.attrib]`